agentstack/cli/cli.py

```diff
@@ -59,18 +47,27 @@ def configure_default_model():
     agentstack_config = ConfigFile()
     if agentstack_config.default_model:
         log.debug("Using default model from project config.")
-        return  # Default model already set
```

Comment:
we're very intentional with which comments stay in and which just bloat the package size. i'd rather not delete these